### PR TITLE
openssl: display a domain's cert's start and end dates

### PR DESCRIPTION
**Pull request by andrewlouis93**
* Pretty great use case for `openssl` - to check the start and end date of a domain's certificate. 
* Also demonstrates how you can pipe a cert to `openssl x509` to read particular fields from it - which none of the existing commands seem to do.

### CODE REVIEW

File: pages/common/openssl.md [hunk 1]
```diff
@@ -14,6 +14,10 @@
 
 `openssl x509 -req -days {{days}} -in {{filename.csr}} -signkey {{filename.key}} -out {{filename.crt}}`
 
+- Only display a domain's certificate's start and end dates:
```
Comment (sbrl) [inline pages/common/openssl.md]: Perhaps "Display the start and end dates for a domain's certificate" would read better?

Comment (andrewlouis93) [reply pages/common/openssl.md]: 👍 that does read a bit better. Updated, thanks!

Comment (agnivade) [reply pages/common/openssl.md]: Should it be expiry date instead of end date ? Maybe that fits better for a cert's context.

**Comment by sbrl** [comment]
Cool! Looks good to me now :smiley_cat:

**Comment by andrewlouis93** [comment]
@sbrl i don't have write access to this repo, could you merge in this PR?

**Comment by agnivade** [comment]
Hi @andrewlouis93 - I posted a minor comment [here](https://github.com/tldr-pages/tldr/pull/1486#discussion_r138434056). Let me know what you think about it.

We usually merge PRs when atleast 2 maintainers have approved the changes.

File: pages/common/openssl.md (hunk 1 above)
Comment (andrewlouis93) [reply pages/common/openssl.md]: Yeah, sure! I think that flows better.

**Comment by andrewlouis93** [comment]
@agnivade updated :)

**Comment by sbrl** [comment]
Looks ok to me! Good idea, @agnivade.

@andrewlouis93 We have a policy of 2 approvals on a PR by 2 different members with merge access. i.e. the first one to review approves it, and the second person to approve merges the request.

Ping @waldyrious.

**Comment by agnivade** [comment]
Thanks

**Comment by waldyrious** [comment]
Sorry I'm late to the party, but I suspect the sequence of examples would work better with this example swapped with the next one, which seems to be a simpler version. This way it would build upon the previous example and avoid introducing a lot of new stuff at once. If you guys agree, I can make the change myself.

**Pull request merged**